pkg/hooks/audit_logger_sqlite.go

```diff
@@ -0,0 +1,184 @@
+package hooks
+
+import (
+	"database/sql"
+	"fmt"
+
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/util"
+	"github.com/square/p2/pkg/util/param"
+)
+
+var auditLogTTL = param.Int("audit_log_sqlite_ttl", 259200)
```

Comment:
I see Github thinks https://github.com/square/p2/pull/823#discussion_r113801096 is out of date but I say it still applies!